include/NAS2D/Resources/Image.h

```diff
@@ -40,10 +40,10 @@ namespace NAS2D {
 class Image: public Resource
 {
 public:
-	Image(const std::string& filePath);
+    Image();
+    Image(const std::string& filePath);
 	Image(void* buffer, int bytesPerPixel, int width, int height);
 	Image(int width, int height);
```

Comment:
The indentation in this file looks inconsistent.

Reply:
Probably a tabs vs spaces and clang-tidy collision.